website/identifiers/clients/base.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+import requests
+
+from framework.exceptions import HTTPError
+from website import settings
+
+
+class AbstractIndentifierClient(object):
+
+    def __init__(self, base_url, prefix):
```

Comment:
This gets overriden by `CrossRefClient`, which is a bit of a code smell. Perhaps this doesn't belong on the abstract class?

Reply:
Also, `base_url` should not vary across instances, correct? If so, it would make sense for `base_url` to be a class variable.

```python
class Abstract...:
    base_url = None

class Datacite...(Abstract...):
    base_url = 'https://datacite...'

```

If you need to override `base_url`, e.g. for tests, I suppose you could do something like

```python
class Abstract...:
    default_base_url = None

    def __init__(self, base_url=None):
        self.base_url = base_url or self.default_base_url
```

But I'm not sure that's necessary.